src/dlnasettings.html

```diff
@@ -10,11 +10,6 @@
                         <h2 class="sectionTitle">${TabSettings}</h2>
                         <a is="emby-linkbutton" class="raised button-alt headerHelpButton" target="_blank" href="https://web.archive.org/web/20181216120305/https://github.com/MediaBrowser/Wiki/wiki/Dlna-settings">${Help}</a>
                     </div>
-
```

Comment:
hmm, why remove?

Reply:
so more code can fit on my screen :p there are a lot of newlines

Reply:
I was referring to the whole removed block, it just happened to start with newline

1.

Reply:
Ah that one was in the description.

> The DLNA page had dead code referencing the streaming bitrate limit, which is currently in the playback section.